src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDiscountDays = 7;
+  const shortTermDiscountDays = 3;
+  const carPricePerDay = 40;
+  const carPricePerRent = days * carPricePerDay;
+
+  if (days >= longTermDiscountDays) {
+    return carPricePerRent - 50;
+  } else if (days >= shortTermDiscountDays && days < longTermDiscountDays) {
+    return carPricePerRent - 20;
```

Comment:
```suggestion
  if (days >= longTermDiscountDays) {
    return carPricePerRent - 50;
  }
  
  if (days >= shortTermDiscountDays && days < longTermDiscountDays) {
    return carPricePerRent - 20;
```
`else` is redundant here as the code will not go further after return anyway